Formula/wxmac@3.0.rb

```diff
@@ -0,0 +1,70 @@
+class WxmacAT30 < Formula
+  desc "Cross-platform C++ GUI toolkit (wxWidgets for macOS) - Stable Release"
+  homepage "https://www.wxwidgets.org"
+  url "https://github.com/wxWidgets/wxWidgets/releases/download/v3.0.5.1/wxWidgets-3.0.5.1.tar.bz2"
+  sha256 "440f6e73cf5afb2cbf9af10cec8da6cdd3d3998d527598a53db87099524ac807"
+  license "wxWindows"
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+\.\d*[02468](?:\.\d+)*)$/i)
```

Comment:
```suggestion
    regex(/^v?(3\.0(?:\.\d+)*)$/i)
```
Since this is `wxmac@3.0` specifically, and not `wxmac-stable`, should we restrict matching to `3.0.x`? Can be done in a separate PR if you agree and want to avoid rerunning CI for a syntax change.

Reply:
I wonder if we should just make this `wxmac-stable` so that we could always just have their development and stable branches as two formulae... We don't actually care about the `3.0.x` branch, I think.

Reply:
That seems like a good option to me since we have formulae pairs like `jenkins` and `jenkins-lts`.

Reply:
I've made `wxmac-stable` keg-only, as I suspect there will be situations where users will need to have both installed (cf. `ilmbase` -> `imath`)

Reply:
Actually, it looks like the wxWidgets developers mostly support having multiple versions installed in parallel. All the libraries are explicitly versioned and the headers live in versioned subdirectories, for example. We don't need to make `wxmac-stable` keg-only after moving a few files around.

Reply:
Since we went back to `wxmac@3.0`, just leaving a note here that we should change the `livecheck` regex. Thanks for your patience and all the work here.

Reply:
Hmm, thinking about it some more, maybe we want to keep it as is? We still need a way to track the stable version, especially if, say, 3.2.x is released after 3.3.y.

Reply:
That makes sense, but that would deviate from our usual standards for `livecheck` in versioned formulae (with this specifically being `3.0` 😅). Well I don't feel too strongly anyway and we can handle this in follow up PRs, if we want to change this behaviour.

Reply:
Yes, my thinking is that unless we really need `wxmac@3.0` to stick around we'll just do a formula rename from `wxmac@3.0` to `wxmac@3.2` so that anyone who needs the stable version can stay on it easily...